linetools/lists/parse.py

```diff
@@ -0,0 +1,353 @@
+                        int(tbl[count]['wrest']))
+                    #xdb.set_trace()
+                    # f
+                    try:
+                        tbl[count]['f'] = float(line[79:89])
+                    except ValueError:
+                        continue # Skip ones without f-value
+                    # Ej, Ek
+                    tbl[count]['Ej'] = float(line[29:38]) #/ u.cm
+                    tbl[count]['Ek'] = float(line[40:50]) #/ u.cm
+                    # A
+                    try:
+                        tbl[count]['A'] = float(line[59:68]) #/ u.s
+                    except ValueError:
+                        pass
+                    # gamma
+                    try:
+                        tbl[count]['gamma'] = float(line[69:79]) #/ u.s
+                    except ValueError:
+                        pass
+                    # gl, gu
+                    tbl[count]['gj'] = int(line[52:54])
+                    tbl[count]['gk'] = int(line[56:58])           
+                    # Ex
+                    #all_dict[count]['Ex'] = 0.  # Zero out units (for Table)
+
+                    # Check
+                    #print(tbl[count])
+                    #xdb.set_trace()
+
+                    # Increment
+                    count += 1
+        # Trim
+        data = tbl[0:count]
+
+        # Finish up
+        #data['A'].unit = 1/u.s
+        #data['gamma'].unit = 1/u.s
+        #data['wrest'].unit = u.AA
+        #data['Ex'].unit = 1/u.cm
+        #data['Ej'].unit = 1/u.cm
+        #data['Ek'].unit = 1/u.cm
+
+        # Last
+        data['group'] = 1
+        data['Ref'] = 'Morton2003'
+        data['mol'] = ''
+
+    # Return
+    return data
+
+def mkvot_morton03(do_this=False, outfil=None):
+    '''Used to generate a VO Table for the Morton2003 paper
+    Only intended for builder usage (1.5Mb file; gzip FITS is 119kb)
+    '''
+    if not do_this:
+        print('mkvot_morton03: It is very unlikely you want to do this')
+        print('mkvot_morton03: Returning...')
+        return
+    # Read Morton2003
+    m03 = parse_morton03()
+
+    # Write
+    if outfil is None:
+        outfil = lt_path + '/data/lines/morton03_table2.vot'
+    m03.write(outfil, format='votable')
+
+
+def roman_to_number(val):
+    '''Convert simple Roman numerals to Arabic
+
+    Parameters:
+    -------------
+    val: str or unicoce
+      Roman numeral for conversion
+    Returns:
+    ------------
+    Number
+    '''
+    if val.strip() == 'I':
+        return 1
+    elif val.strip() == 'II':
+        return 2
+    elif val.strip() == 'III':
+        return 3
+    elif val.strip() == 'IV':
+        return 4
+    elif val.strip() == 'V':
+        return 5
+    elif val.strip() == 'VI':
+        return 6
+    elif val.strip() == 'VII':
+        return 7
+    elif val.strip() == 'VIII':
+        return 8
+    else:
+        raise ValueError('Not setup for {:s}'.format(val)) 
+
+###########################################
```

Comment:
I would replace this function with a dictionary:

```roman_to_number = dict(I=0, II=1, III=2, IV=3, V=4, VI=5, VII=6, VIII=7, IX=8, X=9)```

Index it with `roman_to_number[val.strip()]` if you really need to remove whitespace.